include/vapor/ArbitrarilyOrientedRegularGrid.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <ostream>
+#include <vector>
+#include <vapor/common.h>
+#include <vapor/RegularGrid.h>
+#include <glm/glm.hpp>
+
+// clang-format off
+
+struct planeDescription {
```

Comment:
Please document the public members of this class

Reply:
Fixed.

Reply:
@clyne - I'm not sure why this code is missing the (outdated) label.  You can find the changes on the [current commit hash of this branch](https://github.com/NCAR/VAPOR/pull/2974/commits/7bb2296c47ce030971e08910ed522cfb931c78aa).